### PR TITLE
Feat: Avtar component  added

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
1.avatar component css code is written in avatar.css file.
2.avatar component is imported in doc.css file.
3.docs.css fle consists of styling specific to documentation page.
4.html code for all the components I will  write in doc.html file.
5.component library start from index.html page.
6. navbar.css is file written for navigation bar.
7. navbar I make responsive. for other things on doc.html page i will add responsive ness after words.

Please review it and suggest your valuable inputs for merging this branch to dev.

**Screenshots**
avatar2
avatar1

### CODE REVIEW

File: documentation/doc.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    <!-- Navigation Bar -->
+    <nav>
+        <div class="nav-logo">SARATHI</div>
+        <input type="checkbox" id="check-box-click">
+        <label for="check-box-click" class="nav-menu-btn">
+            <i class="fas fa-align-justify"></i>
+        </label>
+        <ul>
+            <li><a class="nav-active" href="#">Home</a></li>
+            <li><a href="#">Gallary</a></li>
```
Comment (sks-21) [inline documentation/doc.html]: Typo, Gallery

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge

File: documentation/doc.html (hunk 1 above)
Comment (kotalwar-krishna) [reply documentation/doc.html]: Thanku, I will do.

**Review by JantuDeb — APPROVED** [review]
CSS for the avatar component looks fine, you can merge it.

**Comment by kotalwar-krishna** [comment]
> CSS for the avatar component looks fine, you can merge it.

Thnaku

**Pull request merged**